yt_dlp/extractor/thisvid.py

```diff
@@ -1,97 +0,0 @@
-# coding: utf-8
-from __future__ import unicode_literals
-import re
-
-from .common import InfoExtractor
-
-
-class ThisVidIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?thisvid\.com/(?P<type>videos|embed)/(?P<id>[A-Za-z0-9-]+/?)'
-    _TESTS = [{
-        'url': 'https://thisvid.com/videos/french-boy-pantsed/',
-        'md5': '3397979512c682f6b85b3b04989df224',
-        'info_dict': {
-            'id': '2400174',
-            'ext': 'mp4',
-            'title': 'French Boy Pantsed',
-            'thumbnail': 'https://media.thisvid.com/contents/videos_screenshots/2400000/2400174/preview.mp4.jpg',
-            'age_limit': 18,
-        }
-    }, {
-        'url': 'https://thisvid.com/embed/2400174/',
-        'md5': '3397979512c682f6b85b3b04989df224',
-        'info_dict': {
-            'id': '2400174',
-            'ext': 'mp4',
-            'title': 'French Boy Pantsed',
-            'thumbnail': 'https://media.thisvid.com/contents/videos_screenshots/2400000/2400174/preview.mp4.jpg',
-            'age_limit': 18,
```

Comment:
The `age_limit` information is lost when using generic. Make this a thin wrapper instead of deleting

Reply:
`age_limit` was never really calculated, it was just hard-coded to 18+ every time. I don't think a `if url.startswith('https://thisvid.com')` is really useful.